platform-coop-toolkit/pwa-experiments
# Add Service Worker (resolves #3)

## Description

Adds a Service Worker which caches CSS and JS, and provides an offline version of the page.

## Steps to test

1. Visit the site at https://pwa-experiments.netlify.com.
2. Turn off your WIFI.
3. Reload the page.

**Expected behavior:** Offline mode!

## Additional information

Largely inspired by techniques in [_Going Offline_](https://abookapart.com/products/going-offline).

## Related issues

- Resolves #3.
